Tangem/Modules/SingleTokenBase/SingleTokenBaseViewModel.swift

```diff
@@ -445,6 +445,22 @@ extension SingleTokenBaseViewModel {
         }
     }
 
+    private func isBuyDisabled() -> Bool {
+        if FeatureProvider.isAvailable(.onramp) {
+            if walletModel.isCustom {
+                return true
+            }
+
+            return expressAvailabilityProvider.canOnramp(tokenItem: walletModel.tokenItem)
```

Comment:
Получается только надо обратно, раз disabled 
```suggestion
            return !expressAvailabilityProvider.canOnramp(tokenItem: walletModel.tokenItem)
```